css/airbnb.css

```
@@ -617,6 +616,321 @@ h2{color:black;
+.guest-title{
+    grid-area:2/1/3/2;
+    background-color: rgb(243, 240, 240);
+    padding-top: 4%;
+    padding-bottom: 4%;
+}
+.guest-info1{
+    grid-area:3/1/4/2;
+    background-color: rgb(243, 240, 240);
+    border-top: solid rgb(207, 207, 207) 2px;
+    padding-top: 4%;
+    padding-bottom: 4%;
+}
+.guest-info2{
+    grid-area:4/1/5/2;
+    background-color: rgb(243, 240, 240);
+    border-top: solid rgb(207, 207, 207) 2px;
+    padding-top: 4%;
+    padding-bottom: 4%;
+}
+.guest-info3{
+    grid-area:5/1/6/2;
+    background-color: rgb(243, 240, 240);
+    border-top: solid rgb(207, 207, 207) 2px;
+    padding-top: 4%;
+    padding-bottom: 4%;
+}
+.host-title{
+    grid-area:2/2/3/3;
+    background-color: rgb(243, 240, 240);
+    padding-top: 4%;
+    padding-bottom: 4%;
+}
+.host-info1{
+    grid-area:3/2/4/3;
+    background-color: rgb(243, 240, 240);
+    border-top: solid rgb(207, 207, 207) 2px;
+    padding-top: 4%;
+    padding-bottom: 4%;
+}
+.host-info2{
+    grid-area:4/2/5/3;
+    background-color: rgb(243, 240, 240);
+    border-top: solid rgb(207, 207, 207) 2px;
+    padding-top: 4%;
+    padding-bottom: 4%;
+}
+.host-info3{
+    grid-area:5/2/6/3;
+    background-color: rgb(243, 240, 240);
+    border-top: solid rgb(207, 207, 207) 2px;
+    padding-top: 4%;
+    padding-bottom: 4%;
+}
+.corona-title{
+    grid-area:2/3/3/4;
+    background-color: rgb(243, 240, 240);
+    padding-top: 4%;
+    padding-bottom: 4%;
+}
+.corona-info1{
+    grid-area:3/3/4/4;
+    background-color: rgb(243, 240, 240);
+    border-top: solid rgb(207, 207, 207) 2px;
+    padding-top: 4%;
+    padding-bottom: 4%;
+}
+.corona-info2{
+    grid-area:4/3/5/4;
+    background-color: rgb(243, 240, 240);
+    border-top: solid rgb(207, 207, 207) 2px;
+    padding-top: 4%;
+    padding-bottom: 4%;
+}
+.corona-info3{
+    grid-area:5/3/6/4;
+    background-color: rgb(243, 240, 240);
+    border-top: solid rgb(207, 207, 207) 2px;
+    padding-top: 4%;
+    padding-bottom: 4%;
+}
+.etc-title{
+    grid-area:2/4/3/5;
+    background-color: rgb(243, 240, 240);
+    padding-top: 4%;
+    padding-bottom: 4%;
+}
+.etc-info1{
+    grid-area:3/4/4/5;
+    background-color: rgb(243, 240, 240);
+    border-top: solid rgb(207, 207, 207) 2px;
+    padding-top: 4%;
+    padding-bottom: 4%;
+}
+.etc-info2{
+    grid-area:4/4/5/5;
+    background-color: rgb(243, 240, 240);
+    border-top: solid rgb(207, 207, 207) 2px;
+    padding-top: 4%;
+    padding-bottom: 4%;
+}
+.etc-info3{
+    grid-area:5/4/6/5;
+    background-color: rgb(243, 240, 240);
+    padding-top: 4%;
+    padding-bottom: 4%;
+}
+#info-text1{
+    font-size:20px;
+    color:black;
+
+}
+#info-text2{
+    font-size:18px;
+    color: black;
+}
+#info-text3{
+    font-size:12px;
+    color: rgb(107, 106, 106);
+
+}
+h1{
+    color:white;
+    font-size: 35px;
+    font-weight: bold;
+}
+h2{color:black;
+    font-size:22px;
+    font-weight:bold;
+}
+
+a {text-decoration: none;}
```

css 컨벤션을 통일 시켜주세요 ^^

```suggestion
h2 {
    color:black;
    font-size:22px;
    font-weight:bold;
}

a {
    text-decoration: none;
}
```